stories/13-Collection.stories.tsx

```diff
@@ -381,178 +484,180 @@ export const AllPost = () => {
   );
 };
 
-enum DisplayType {
-  NEW,
-  UPDATED,
-  ALL,
-}
+// This is so outdated that it's totally broken and should just completely get reworked when updating to the new story format
```

Comment:
Honestly I think it might just be better to completely get rid of the Collections story. I did this as an attempt to figure out how to do timeline/gallery, but as far as I can tell there's nothing here that isn't tested/done better somewhere else, and it just adds complexity.

Reply:
Okay, I will kill it when I do the file reorganization